kratos/geometries/quadrilateral_3d_4.h

```diff
@@ -419,41 +419,41 @@ template<class TPointType> class Quadrilateral3D4
         {
            area += temp[i] * integration_points[i].Weight();
         }
-        
+
         return area;
-        
+
 //         // 24/01/2014 - Massimo Petracca

```

Comment:
Maybe if this code is not going to be used we can remove it

Reply:
? I see no unused code

Reply:
425 and following , deactivate the whitespace filter and you will see

Reply:
Ok, but maybe in another PR, has nothing to do with this one...